File: PersonalDoctors/ClientApp/src/app/patient.ts
export class Patient {
    constructor(
        public id?: number,
        public name?: string,
        public surname?: string,
        public patronymic?: string,
        public gender?: string,
        public email?: string,
        public numberpolicy?: string,
        public numberpassport?: string,
        public phone?: string,
        public photourl?: string,
        public residenceaddress?: string,
        public placeofresidence?: string) { }
}

File: PersonalDoctors/Models/Patient.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace PersonalDoctors.Models
{
    public class Patient
    {
        public int id { get; set; }
        public string name { get; set; }
        public string surname { get; set; }
        public string patronymic { get; set; }
        public string gender { get; set; }
        public string email { get; set; }
        public string numberpolicy { get; set; }
        public string numberpassport { get; set; }
        public string phone { get; set; }
        public string photourl { get; set; }
        public string residenceaddress { get; set; }
        public string placeofresidence { get; set; }

    }
}
